sources/packages/api/src/esdc-integration/e-cert-integration/e-cert-full-time-integration/e-cert-files/e-cert-file-record.ts

```diff
@@ -194,8 +202,8 @@ export class ECertFullTimeFileRecord extends ECertFileRecord {
     record.appendWithEndFiller(this.addressLine1, 40, SPACE_FILLER);
     record.appendWithEndFiller(this.addressLine2 ?? "", 40, SPACE_FILLER);
     record.appendWithEndFiller(this.city, 25, SPACE_FILLER);
-    record.repeatAppend(SPACE_FILLER, 4); // Province, optional, not provided.
-    record.repeatAppend(SPACE_FILLER, 16); // Postal code, optional, not provided.
+    record.appendWithEndFiller(this.provinceState ?? "", 4, SPACE_FILLER);
+    record.appendWithEndFiller(this.postalCode ?? "", 16, SPACE_FILLER);
```

Comment:
`postalCode` is not optional

Reply:
At this level, it can. From the file perspective, we can avoid sending it from countries different from Canada. Not a final decision on this but the properties can reflect what the e-Cert supports.